CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## 2.17.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.2.1890**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for ``enable_monitor_gateway_subnets`` and ``monitor_exclude_list`` in **aviatrix_gateway**
+2. Implemented support for ``idle_timeout`` and ``renegotiation_interval`` in **aviatrix_gateway**
```

Comment:
2. Implemented support for managing Aviatrix VPN timeout configurations through `idle_timeout` and `renegotiation_interval` in **aviatrix_gateway**